### ISSUE TITLE
NVDA does not read value in text box in IE 11

### ISSUE THREAD

**Issue by dmitriyyepishin** [nvaccess/nvda#8748]
### Steps to reproduce:
1. Go to https://dev.office.com/fabric-js/Components/TextField/TextField.html
2. Scroll down to "Default TextField"
3. Enter "hello" in text field
4. Move mouse over "hello" that was entered
5. Observe that NVDA does not read "hello"

_NOTE: This works fine on Chrome. The problem occurs in IE 11._

### Actual behavior:
NVDA does not read the text in text field

### Expected behavior:
NVDA should read the text

**Comment by Adriani90** [comment]
Support for IE ended completely in 2022, and the browser is automatically removed from machines that still have it after an Edge update in February 2023. I don't expect any further updates regarding accessibility. If you still have this issue in other browsers, please open a new updated issue with coresponding steps to reproduce and a minimal test case.
I am closing this as abandoned.
